hera_cal/lstbin.py

```diff
@@ -542,6 +549,7 @@ def lst_bin_arg_parser():
     a.add_argument("--outdir", default=None, type=str, help="directory for writing output")
     a.add_argument("--overwrite", default=False, action='store_true', help="overwrite output files")
     a.add_argument("--lst_start", type=float, default=None, help="starting LST for binner as it sweeps across 2pi LST. Default is first LST of first file.")
+    a.add_argument("--weight_by_nsamples", default=True, action='store_true', help="Weight by nsamples during LST binning. If set to False, weight by flags only. Default True.")
```

Comment:
I don't *think* this will work. As far as I know, there's no way to set the flag to False on the command line. So I think you'll need to use something like `--weight-by-flags-only`

Reply:
Gotcha, thanks! I will switch all the options to `weight_by_flags_only`.